src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -573,7 +571,7 @@
 type: command
 short-summary: Create an Azure Active Directory user.
 parameters:
-  - name: --force-change-password-next-login
+  - name: --force-change-password-next-sign-in
```

Comment:
This breaking change is undocumented in Graph doc (https://github.com/microsoftgraph/microsoft-graph-docs/issues/16819).